### DISCUSSION TITLE
homelab

### DISCUSSION THREAD

**Discussion by giscus** [insertish/Portfolio#17] [Comments]
# homelab

Information about my home infrastructure.

https://insrt.uk/homelab

**Comment by FanMclaine** [comment]
imagine ub*ntu :trollface:

**Reply by ghost** [reply]
its a server its ok

**Comment by sussycatgirl** [comment]
:trollface:

**Comment by vmfunc** [comment]
netscape

**Comment by ghost** [comment]
wacky ass homelab

**Reply by sussycatgirl** [reply]
https://user-images.githubusercontent.com/26145882/194108062-d691b70d-8adf-4d7d-a911-9db63ca37ca4.mp4

**Reply by ghost** [reply]
what in tarnation
